serverless invoke local fails with java11 runtime

# Bug Report
When creating and running the default template for aws-java-maven, I changed the default runtime from java8 to java11 (As AWS Lambda supports this) and am getting failures.
## Description

<!-- Please use https://forum.serverless.com, StackOverflow or other forums for Q&A -->
<!-- Please answer ALL the question below. Otherwise we probably have to close the issue due to missing information -->

1. What did you do?
Run `serverless create -t aws-java-macen -n test-project`
Edit `serverless.yml` to change `runtime: java8` to `runtime: java11`
Run `mvn compile package`
Run `serverless invoke local -f hello` produced below error

```bash
Serverless: Packaging service...
Serverless: Building Docker image...
Serverless: Downloading base Docker image...

  Error --------------------------------------------------

  Error: EEXIST: file already exists, mkdir 'C:\<user>\<projectpath>\.serverless\invokeLocal\artifact\META-INF'
      at Object.mkdirSync (fs.js:823:3)
      at Function.sync (C:\<user>\AppData\Roaming\npm\node_modules\serverless\node_modules\mkdirp\index.js:71:13)
      at C:\<user>\AppData\Roaming\npm\node_modules\serverless\lib\plugins\aws\invokeLocal\index.js:347:22
      at runMicrotasks (<anonymous>)
      at runNextTicks (internal/process/task_queues.js:58:5)
      at processImmediate (internal/timers.js:412:9)
      at process.topLevelDomainCallback (domain.js:131:23)

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com

  Your Environment Information ---------------------------
     Operating System:          win32
     Node Version:              12.13.0
     Framework Version:         1.61.3
     Plugin Version:            3.3.0
     SDK Version:               2.3.0
     Components Core Version:   1.1.2
     Components CLI Version:    1.4.0
```
Adding: `mvn -v`
```bash
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: C:\Users\sesa554024\Downloads\apache-maven-3.6.3\bin\..
Java version: 11.0.6, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-11.0.6
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```

1. What happened?
Attempted to download docker image
1. What should've happened?
Run similar to java8 invoke local
1. What's the content of your `serverless.yml` file?
default java template with runtime modified
```yaml
service: test-project
provider:
  name: aws
  runtime: java11

package:
  artifact: target/hello-dev.jar

functions:
  hello:
    handler: com.serverless.Handler
```
1. What's the output you get when you use the `SLS_DEBUG=*` environment variable (e.g. `SLS_DEBUG=* serverless deploy`)
`SLS_DEBUG=* serverless invoke local -f hello`
```bash
SLS_DEBUG=* : The term 'SLS_DEBUG=*' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included,
verify that the path is correct and try again.
At line:1 char:1
+ SLS_DEBUG=* serverless invoke local -f hello
+ ~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (SLS_DEBUG=*:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```
Similar or dependent issues:
Haven't seen any

Found SLS_DEBUG=* fix for windows

```bash
Serverless: Load command interactiveCli
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command create
Serverless: Load command install
Serverless: Load command package
Serverless: Load command deploy
Serverless: Load command deploy:function
Serverless: Load command deploy:list
Serverless: Load command deploy:list:functions
Serverless: Load command invoke
Serverless: Load command invoke:local
Serverless: Load command info
Serverless: Load command logs
Serverless: Load command metrics
Serverless: Load command print
Serverless: Load command remove
Serverless: Load command rollback
Serverless: Load command rollback:function
Serverless: Load command slstats
Serverless: Load command plugin
Serverless: Load command plugin
Serverless: Load command plugin:install
Serverless: Load command plugin
Serverless: Load command plugin:uninstall
Serverless: Load command plugin
Serverless: Load command plugin:list
Serverless: Load command plugin
Serverless: Load command plugin:search
Serverless: Load command config
Serverless: Load command config:credentials
Serverless: Load command rollback
Serverless: Load command rollback:function
Serverless: Load command upgrade
Serverless: Load command uninstall
Serverless: Load command login
Serverless: Load command logout
Serverless: Load command generate-event
Serverless: Load command test
Serverless: Load command dashboard
Serverless: Load command output
Serverless: Load command output:get
Serverless: Load command output:list
Serverless: Load command param
Serverless: Load command param:get
Serverless: Load command param:list
Serverless: Invoke invoke:local
Serverless: In order to get human-readable output, please implement "toString()" method of your "ApiGatewayResponse" object.
java.lang.reflect.InvocationTargetException

        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at com.serverless.InvokeBridge.invoke(InvokeBridge.java:95)
        at com.serverless.InvokeBridge.<init>(InvokeBridge.java:39)
        at com.serverless.InvokeBridge.main(InvokeBridge.java:150)
Caused by: java.lang.NoSuchMethodError: com.amazonaws.services.lambda.runtime.LambdaLogger.log([B)V
        at com.amazonaws.services.lambda.runtime.log4j2.LambdaAppender.append(LambdaAppender.java:74)
        at org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:156)
        at org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:129)
        at org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:120)
        at org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:84)
        at org.apache.logging.log4j.core.config.LoggerConfig.callAppenders(LoggerConfig.java:448)
        at org.apache.logging.log4j.core.config.LoggerConfig.processLogEvent(LoggerConfig.java:433)
        at org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:417)
        at org.apache.logging.log4j.core.config.LoggerConfig.log(LoggerConfig.java:403)
        at org.apache.logging.log4j.core.config.AwaitCompletionReliabilityStrategy.log(AwaitCompletionReliabilityStrategy.java:63)
        at org.apache.logging.log4j.core.Logger.logMessage(Logger.java:146)
        at org.apache.logging.log4j.spi.AbstractLogger.logMessageSafely(AbstractLogger.java:2091)
        at org.apache.logging.log4j.spi.AbstractLogger.logMessage(AbstractLogger.java:2005)
        at org.apache.logging.log4j.spi.AbstractLogger.logIfEnabled(AbstractLogger.java:1876)
        at org.apache.logging.log4j.spi.AbstractLogger.info(AbstractLogger.java:1421)
        at com.serverless.Handler.handleRequest(Handler.java:18)
        ... 7 more
```

> Caused by: java.lang.NoSuchMethodError: com.amazonaws.services.lambda.runtime.LambdaLogger.log([B)V
>         at com.amazonaws.services.lambda.runtime.log4j2.LambdaAppender.append(LambdaAppender.java:74)

@thecpdubguy I think downgrading aws-lambda-java-log4j2 to 1.0.0 in pom.xml fixes the issue. See #5155 for more details.

--- a/lib/plugins/aws/invokeLocal/index.js
+++ b/lib/plugins/aws/invokeLocal/index.js
@@ -201,7 +201,7 @@ class AwsInvokeLocal {
       );
     }
 
-    if (runtime === 'java8') {
+    if (['java8', 'java11'].includes(runtime)) {
       const className = handler.split('::')[0];
       const handlerName = handler.split('::')[1] || 'handleRequest';
       return this.invokeLocalJava(

